4.0-MIGRATION-GUIDE.md

```diff
@@ -44,7 +44,7 @@ Non-breaking:
 ## Theming Changes
 
 We have promoted all of the theming functionality out of Canvas Kit Labs.
-[It now lives in `@workday/canvas-kit-react-common](https://github.com/Workday/canvas-kit/tree/prerelease/v4/modules/common/react/lib/theming).
+[It now lives in `@workday/canvas-kit-react-common`](https://github.com/Workday/canvas-kit/tree/prerelease/v4/modules/common/react/lib/theming).
```

Comment:
Nit: should the link just be for `@workday/canvas-kit-react-common` instead of including "It now lives in"? Then again, you already fixed this sentence explicitly, so maybe the intent _is_ to make the entire sentence a link.